man/systemd.netdev.xml

```diff
@@ -1268,6 +1268,27 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>AdActorSysPrio=</varname></term>
```

Comment:
Please, let's not needlessly abbreviate. Unless some abbreviation is very well established let's write things out, it makes things more self-explanatory for readers, and we generally do so across the trees.

Hence, `AdActorSystemPriority` maybe?

Reply:
In principle the names are the same as they are in the sysfs (`/sys/class/net/$IFNAME/bonding`), besides the CamelCase adaptation. This one is originally ad_actor_sys_prio. The same names are accepted from iproute2 (see `ip link help bond`). None the less I would not mind changing them.

Reply:
I'd really prefer the mor verbose naming in this case

Reply:
Great! Updates are pushed.